invertedai/api_resources.py

```diff
@@ -30,6 +34,10 @@
 
 TIMEOUT = 10
 
+mock_api = bool(os.environ.get("IAI_MOCK_API", False))
```

Comment:
small suggestions.. strtobool is available to support slightly more cases
```
from distutils.util import strtobool
strtobool(os.environ.get("IAI_MOCK_API", "false")
```

True values are y, yes, t, true, on and 1; false values are n, no, f, false, off and 0

Reply:
done